### PR TITLE
tests: add DFL mock sysfs for N3000 and D5005

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: framework/platform/fpga_hw.cpp [hunk 1]
```diff
@@ -127,6 +127,44 @@ const char *vc_mdata =
...
+   "afu-image":
+    {"clock-frequency-high": 312,
+     "clock-frequency-low": 156,
+     "interface-uuid": "a5d72a3c-c8b0-4939-912c-f715e5dc10ca",
+     "magic-no": 488605312,
+     "accelerator-clusters":
+      [
+        {
+          "total-contexts": 1,
+          "name": "nlb3",
```
Comment (tswhison) [inline framework/platform/fpga_hw.cpp:141]: The tarball is named NLB0, but this says nlb3.

File: framework/platform/fpga_hw.cpp [hunk 2]
```diff
@@ -127,6 +127,44 @@ const char *vc_mdata =
...
+   "afu-image":
+    {"clock-frequency-high": 312,
+     "clock-frequency-low": 156,
+     "interface-uuid": "9346116d-a52d-5ca8-b06a-a9a389ef7c8d",
+     "magic-no": 488605312,
+     "accelerator-clusters":
+      [
+        {
+          "total-contexts": 1,
+          "name": "nlb3",
```
Comment (tswhison) [inline framework/platform/fpga_hw.cpp:160]: Likewise, the tarball says nlb0, but this says nlb3.

File: framework/platform/fpga_hw.cpp (hunk 1 above)
Comment (anandaravuri) [reply framework/platform/fpga_hw.cpp]: fixed

File: framework/platform/fpga_hw.cpp (hunk 2 above)
Comment (anandaravuri) [reply framework/platform/fpga_hw.cpp]: fixed

**Pull request merged**